src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Page.al

```diff
@@ -0,0 +1,319 @@
+                    begin
+                        AITALTestSuiteMgt.AssistEditTestRunner(Rec);
+                        CurrPage.Update(true);
+                    end;
+                }
+                group(StatusGroup)
+                {
+                    Caption = 'Suite Status';
+                    field(Status; Rec.Status)
+                    {
+                        ToolTip = 'Specifies the status of the test.';
+                        ApplicationArea = All;
+                    }
+                    field(Started; Rec."Started at")
+                    {
+                        ToolTip = 'Specifies when the test was started.';
+                        ApplicationArea = All;
+                    }
+                    field(Version; Rec.Version)
+                    {
+                        ToolTip = 'Specifies the current version of the test run. Log entries will get this version no.';
+                        ApplicationArea = All;
+                        Editable = false;
+                    }
+
+                    field(Tag; Rec.Tag)
+                    {
+                        ToolTip = 'Specifies the tag for a test run. The Tag will be transferred to the log entries and enables comparison between tests.';
+                        ApplicationArea = All;
+                    }
+                }
+            }
+            part(AITTestMethodLines; "AIT Test Method Lines")
+            {
+                ApplicationArea = All;
+                Enabled = Rec.Status <> Rec.Status::Running;
+                SubPageLink = "Test Suite Code" = field("Code"), "Version Filter" = field(Version), "Base Version Filter" = field("Base Version");
+                UpdatePropagation = Both;
+            }
+            group("Latest Run")
+            {
+                Caption = 'Latest Run';
+
+                field("No. of Tests Executed"; Rec."No. of Tests Executed")
+                {
+                    Caption = 'No. of Tests Executed';
+                    ToolTip = 'Specifies the number of tests executed in the current version.';
+                    Editable = false;
+                    ApplicationArea = All;
+                }
+                field("No. of Tests Passed"; Rec."No. of Tests Passed") // TODO: this should be filtered on not empty output
+                {
+                    Caption = 'No. of Tests Passed';
+                    ToolTip = 'Specifies the number of tests passed in the current version.';
+                    Editable = false;
+                    ApplicationArea = All;
+                    Style = Favorable;
+                }
+                field("No. of Tests Failed"; Rec."No. of Tests Executed" - Rec."No. of Tests Passed")
+                {
+                    Editable = false;
+                    ApplicationArea = All;
+                    Style = Unfavorable;
+                    Caption = 'No. of Tests Failed';
```

Comment:
Since the style is fixed, might it be worth adding a BlankZero since zero failed tests is not unfavorable